main.py

```diff
@@ -0,0 +1,60 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+from db.database import SessionLocal, engine
+from db.models import Base
+from schemas import Author, AuthorCreate, Book, BookCreate
+from crud import (
+    create_author,
+    create_book,
+    get_authors,
+    get_author_by_id,
+    get_books,
+    get_books_by_author
+)
+
+app = FastAPI()
+
+Base.metadata.create_all(bind=engine)
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=Author)
+def create_new_author(author: AuthorCreate, db: Session = Depends(get_db)):
+    return create_author(db, author)
+
+
+@app.get("/authors/", response_model=list[Author])
+def read_authors(
+        skip: int = 0, limit: int = 10, db: Session = Depends(get_db)
+):
+    return get_authors(db, skip=skip, limit=limit)
+
+
+@app.get("/authors/{author_id}", response_model=Author)
```

Comment:
```suggestion
@app.get("/authors/{author_id}/", response_model=Author)
```

don't forget to add "/" to the end of the url